src/JabbR-Core/Startup.cs

```diff
@@ -105,8 +118,37 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             app.UseStaticFiles();
             app.UseSignalR();
 
+            //var context = app.ApplicationServices.GetService<JabbrContext>();
+            //AddTestData(context);
            
         }
+
+        private void AddTestData(JabbrContext context)
```

Comment:
The call to this is commented out above - do we need this? If so, I'd throw this in the data project as a FakeData class (something like that) and surround the call to it in the Configure() - not ConfigureServices (thats only for DI) with if (env.IsDevelopment()) { .... }